src/main/java/io/reactivex/Flowable.java

```diff
@@ -9798,10 +9799,13 @@ public final Disposable forEachWhile(final Predicate<? super T> onNext, final Co
      *         unique key value and each of which emits those items from the source Publisher that share that
      *         key value
      * @see <a href="http://reactivex.io/documentation/operators/groupby.html">ReactiveX operators documentation: GroupBy</a>
+     *
+     * @since 2.1.11
```

Comment:
This should fit into 2.1.10, assuming @vanniktech or @artem-zinnatullin approves.